<node
  name="thermal_camera_node"
  type="thermal_camera_node"
  pkg="pandora_vision_hole"
  respawn="False"
  output="$(arg output)"
  ns="$(arg namespace)">

  <rosparam file="$(find pandora_vision_hole)/cfg/thermal_node_cfg/thermal_node_topics.yaml"
  command="load" />

<rosparam file="$(find pandora_vision_hole)/cfg/thermal_node_cfg/thermal_node_matching_variables.yaml"
  command="load" />
</node>

